server/cli/src/refresh_dates.rs

```diff
@@ -50,6 +50,7 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("asset_internal_location", "created_datetime"),
         ("asset_internal_location", "modified_datetime"),
         ("asset_log", "log_datetime"),
+        ("sync_file_reference", "created_datetime"),
```

Comment:
Postgres tests are broken in develop 👀

Reply:
Oops that was me :(
This test always gets me!
Surprised Mark didn't pick it up in code review either as he's usually on postgres...